### PR TITLE
Fix typo and update nuget packages

### PR DESCRIPTION
**Pull request by dsjlee**
updated Microsoft.ML.OnnxRuntimeGenAI.DirectML from v0.2.0-rc7 to v0.2.0. App now has significantly better performance when generating response.

### CODE REVIEW

**Comment by dsjlee** [comment]
@microsoft-github-policy-service agree

File: ChatAppGenAI/ChatAppGenAI.csproj [hunk 1]
```diff
@@ -24,18 +24,12 @@
...
     <PackageReference Include="CommunityToolkit.Mvvm" Version="8.2.2" />
-    <PackageReference Include="Microsoft.ML.OnnxRuntimeGenAI.DirectML" Version="0.2.0-rc7" />
+    <PackageReference Include="Microsoft.ML.OnnxRuntimeGenAI.DirectML" Version="0.2.0" />
     <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.5.240428000" />
-    <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.26031-preview" />
+    <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.26100.1" />
     <Manifest Include="$(ApplicationManifest)" />
   </ItemGroup>
 
-  <ItemGroup>
```
Comment (nmetulev) [inline ChatAppGenAI/ChatAppGenAI.csproj:33]: Should keep this to ensure the model gets copied to the output folder

File: ChatAppGenAI/ModelRunner.cs [hunk 2]
```diff
@@ -13,9 +13,12 @@
 
 namespace ChatAppGenAI
 {
-    public class Phi3Runner : IDisposable
+    public class ModelRunner : IDisposable
     {
-        private readonly string ModelDir = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "phi3");
+        // huggingface-cli download microsoft/Phi-3-mini-4k-instruct-onnx --include directml/* --local-dir .
+        private readonly string ModelDir = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"Models\directml\directml-int4-awq-block-128");
```
Comment (nmetulev) [inline ChatAppGenAI/ModelRunner.cs:19]: If we are changing the folder name for the model, the readme should also be updated

**Review by nmetulev — CHANGES_REQUESTED** [review]
Thank you for this PR. There are few things in here that are also included that need a bit more work before I can merge this:
* you changed the model folder name but didn't update the readme
* you removed the config in the csproj to copy the model to the output

**Comment by dsjlee** [comment]
> Thank you for this PR. There are few things in here that are also included that need a bit more work before I can merge this:
> 
> * you changed the model folder name but didn't update the readme
> * you removed the config in the csproj to copy the model to the output

That's weird, when I made the pull request it only contained two commits on May 25th, not sure how the subsequent 5 commits made on May 28th gets automatically included in pull request retroactively, which are commits for my personal use. If so, then you should reject the pull request.

**Comment by nmetulev** [comment]
Yeah, when you create a PR, all commits from that source branch are included, even those created after the PR was created. Sorry it took so long to review, I didn't have notifications enabled and missed the PR.

I'll close this PR - feel free to submit a new one if not too much trouble.

**Pull request closed without merging**